src/main/java/cosminsechel/driver/BrowserManager.java

```diff
@@ -0,0 +1,15 @@
+package cosminsechel.driver;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class BrowserManager {
+
+    public static ChromeDriver createDriver() {
+        WebDriverManager.chromedriver().setup();
+        return new ChromeDriver();
+    }
+
+    public static void closeChromeDriver(ChromeDriver driver) {
+    }
```

Comment:
You forgot to add driver.quit() inside this method. Don't leave the method body empty. Thanks!